coremltools/optimize/torch/_utils/graph_utils.py

```diff
@@ -105,6 +109,9 @@ def parse_call_function_target(target: _Any) -> str:
     then will use custom parsing code to get the function-name.
     """
 
+    if not _HAS_TORCH_VISION:
+        raise ImportError("torchvision not installed in the python environment")
```

Comment:
Could we use this msg:
https://github.com/apple/coremltools/blob/main/coremltools/_deps/__init__.py#L175

Reply:
Done.